internal/api/server.go

```diff
@@ -160,6 +181,13 @@ func (s *Server) setupRouter() (*gin.Engine, error) {
 		},
 	)
 
+	r.GET(
+		"/metadata",
+		func(c *gin.Context) {
+			c.JSON(200, gin.H{"version": getVersion()})
```

Comment:
Don't use `200`. Look at other route examples. They use the constant `STATUS_OK`. Use that.